final_prep/mod_zero_hero.rb

```diff
@@ -47,38 +63,69 @@
 # assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 #assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+assess_situation(99, announcement, excuse)
+assess_situation(21, announcement, excuse)
+assess_situation(3, announcement, excuse)
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+  :name => "Uncle",
+  :smell => "Smokes",
+  :weight => 240,
+  :citiesDestroyed => ["Reno", "Seattle"],
+  :luckyNumbers => [6 , 9 , 420],
+  :address => {420 => "Donkey" , :NV => 89441}
```

Comment:
For the `address` property on the `scary_monster` hash, the instructions ask for the keys (properties) to be named `number`, `street,` `state`, and `zip` and the values you assign to these keys is left up to you.